uber/model_checks.py

```diff
@@ -120,7 +144,7 @@ def emergency_contact(attendee):
 @validation.Attendee
 @ignore_unassigned_and_placeholders
 def cellphone(attendee):
-    if attendee.cellphone and _invalid_phone_number(attendee.cellphone):
+    if attendee.cellphone and _invalid_phone_number(attendee.cellphone) and not c.COLLECT_FULL_ADDRESS:
```

Comment:
`_invalid_phone_number`, among other things, checks to see if an attendee's phone number matches US-style 10-digit numbers. It skips validating numbers if the `international` checkbox is ticked - however, when we collect the full address, we don't use that checkbox. So this just bypasses the check in those cases.